RabbitMQ: Added note on disabling publisher confirms (#4922)
Particular/docs.particular.net: transports/rabbitmq/connection-settings_publisher-confirms-setting_rabbit_[4,].partial.md

[Publisher confirms](https://www.rabbitmq.com/confirms.html) are enabled by default, but they can be disabled by using the following:

snippet: rabbitmq-config-disablepublisherconfirms

> Note: When publisher confirms are disabled send operations will not verify if relevant exchanges and/or queues exists and will not fail.

Reviewer: ```suggestion
Note: When publisher confirms are disabled, send operations might not fail with an exception when the destination exchanges/queues don't exist.
```
